pkg/loadbalancer/loadbalancer.go

```diff
@@ -397,22 +418,30 @@ func (b *Backend) GetBackendModel() *models.BackendAddress {
 	}
 }
 
-// String returns the L3n4Addr in the "IPv4:Port" format for IPv4 and
-// "[IPv6]:Port" format for IPv6.
+// String returns the L3n4Addr in the "IPv4:Port[/Scope]" format for IPv4 and
+// "[IPv6]:Port[/Scope]" format for IPv6.
 func (a *L3n4Addr) String() string {
+	scope := ""
+	if a.Scope == ScopeInternal {
+		scope = "/i"
+	}
 	if a.IsIPv6() {
-		return fmt.Sprintf("[%s]:%d", a.IP.String(), a.Port)
+		return fmt.Sprintf("[%s]:%d%s", a.IP.String(), a.Port, scope)
 	}
-	return fmt.Sprintf("%s:%d", a.IP.String(), a.Port)
+	return fmt.Sprintf("%s:%d%s", a.IP.String(), a.Port, scope)
 }
 
-// StringWithProtocol returns the L3n4Addr in the "IPv4:Port/Protocol" format
-// for IPv4 and "[IPv6]:Port/Protocol" format for IPv6.
+// StringWithProtocol returns the L3n4Addr in the "IPv4:Port[/Scope]/Protocol"
+// format for IPv4 and "[IPv6]:Port[/Scope]/Protocol" format for IPv6.
```

Comment:
Perhaps would be preferably to have the scope after the `/Protocol`? Usually the protocol is next to the port so it looks weird if we have something in between the two.

Reply:
Agree, changed.